src/utils/formControl.ts

```diff
@@ -0,0 +1,20 @@
+import { Control, RegisterOptions } from 'react-hook-form';
+
+/**
+ * Props that are required for any form control component.
+ */
+export interface FormControlProps {
+
+  /**
+   * The object from the invoking `useForm` hook of React Hook Forms.
+   * Provides API for registering component into needed form.
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  readonly control: Control<any>;
```

Comment:
Can't it be used with `unknown`? not sure

Reply:
No :(